src/FSharpPlus/Operators.fs

```diff
@@ -109,6 +116,7 @@ module Operators =
     /// Sequences two applicatives left-to-right, discarding the value of the second argument.
     let inline (<*  ) (x: '``Applicative<'U>``) (y: '``Applicative<'T>``): '``Applicative<'U>`` = ((fun (k: 'U) (_: 'T) -> k ) <!> x : '``Applicative<'T->'U>``) <*> y
 
+    /// Apply a lifted argument to a lifted function (flipped): arg <**> f
     let inline (<**>) (x: '``Applicative<'T>``) : '``Applicative<'T -> 'U>``->'``Applicative<'U>`` = flip (<*>) x
```

Comment:
I don't find use cases for this operator, I wonder should we deprecate it and free it for other stuff?

Reply:
yes, sounds reasonable

Reply:
https://stackoverflow.com/questions/32426688/are-there-flipped-operators-for-applicative

> Control.Applicative has (<**>), which takes the arguments in the opposite order but is not flip (<*>) because effects are still sequenced from left to right.

Hmm, our implementation **is** flipped.

Reply:
Then it's another reason to deprecate it, at least until v2.0 where we can eventually fix it.